packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -12,7 +12,8 @@ $message-typography: tokens.$body-medium-text-regular;
 $message-margin-top: tokens.$space-300;
 $message-color-default: tokens.$text-secondary-default;
 $message-color-disabled: tokens.$text-secondary-disabled;
-$message-color-error: tokens.$emotion-danger-default;
+$message-color-error: tokens.$emotion-danger-default; // Deprecated (error)
```

Comment:
Please keep the deprecation format the same

Reply:
(and remove this line :))

Reply:
forgot, fixed